### ISSUE TITLE
Installation error - Laravel 7

### ISSUE THREAD

**Issue by yuvrajsab** [wewowweb/laravel-svelte-preset#8]
I'm getting an error while installing this preset.

### What i did
```
composer require wewowweb/laravel-svelte-preset
```

### Getting Error
```
Using version ^0.1.5 for wewowweb/laravel-svelte-preset
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Package operations: 1 install, 0 updates, 0 removals
  - Installing wewowweb/laravel-svelte-preset (v0.1.5): Loading from cache
Writing lock file
Generating optimized autoload files
> Illuminate\Foundation\ComposerScripts::postAutoloadDump
> @php artisan package:discover --ansi

   Error 

  Class 'Illuminate\Foundation\Console\PresetCommand' not found

  at vendor/wewowweb/laravel-svelte-preset/src/LaravelSveltePresetServiceProvider.php:15
    11|      * Bootstrap the application services.
    12|      */
    13|     public function boot()
    14|     {
  > 15|         PresetCommand::macro('svelte', function ($command) {
    16|             SveltePreset::install();
    17|             $command->info('Svelte scaffolding installed successfully.');
    18|             $command->comment('Please run "npm install && npm run dev" to compile your fresh scaffolding.');
    19|         });

  • Database name seems incorrect: You're using the default database name `laravel`. This database does not exist.
    
    Edit the `.env` file and use the correct database name in the `DB_DATABASE` key. 
    https://laravel.com/docs/master/database#configuration

      +8 vendor frames 
  9   [internal]:0
      Illuminate\Foundation\Application::Illuminate\Foundation\{closure}(Object(Wewowweb\LaravelSveltePreset\LaravelSvel
tePresetServiceProvider))

      +5 vendor frames 
  15  artisan:37
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
Script @php artisan package:discover --ansi handling the post-autoload-dump event returned with error code 1

Installation failed, reverting ./composer.json to its original content.
```

It seems like `PresetCommand` is no longer available and the alternative is **new** `UiCommand` from `laravel/ui` package.

**Comment by morpheus7CS** [comment]
Thanks for reporting it. Give me a few minutes to fix it.

Kind regards,
g

**Comment by alesvaupotic** [comment]
Works as advertised, Gal. Thanks for quick fix!

**Comment by morpheus7CS** [comment]
Hey @alesvaupotic,

I actually still can't get it to install properly on Laravel 7.x, actually. I'll work on it some more tonight.

Thanks for the kind words anyway,
g

**Comment by alesvaupotic** [comment]
I am using it right now. 

![image](https://user-images.githubusercontent.com/1660347/75914717-8aac4c00-5e55-11ea-9a7c-5fc91c093883.png)

**Comment by morpheus7CS** [comment]
Hey @yuvrajsab and @alesvaupotic,

I was able to install the dev-master version on a fresh Laravel 7.x installation. I am still having some issues as the tests aren't behaving as they should have. Since there was some "cowboy coding" to get to this stage, I'll tag a new version, but I'm not quite happy with the state that it is in.

I'll keep the issue open & continue working on it tomorrow and see what I come up with, so that it works for all versions as it should.

Kind regards,
g

**Comment by morpheus7CS** [comment]
Hey @yuvrajsab and @alesvaupotic,

I managed to fix the tests today and [v0.1.7](https://github.com/wewowweb/laravel-svelte-preset/releases/tag/v0.1.7) now solves the issue and verifiably works with Laravel 5.8 - 7.x.

Thanks for reporting the issue yesterday.

Kind regards,
g
